flowzone.yml

```diff
@@ -1309,7 +1326,7 @@ jobs:
           if [ ${{ github.run_attempt }} -gt  1 ]; then
             npm --loglevel=verbose --logs-max=0 unpublish ${{ needs.npm_test.outputs.package }}@${{ needs.npm_test.outputs.version_tag }}-$((${{ github.run_attempt }} - 1)) || true
           fi
-          npm --loglevel=verbose --logs-max=0 publish --tag=${{ needs.npm_test.outputs.sha_tag }} "${pack}"
+          npm --loglevel=verbose --logs-max=0 publish --tag=${{ needs.npm_test.outputs.sha_tag }} "${pack}" --access="${{ needs.npm_check.outputs.access }}"
```

Comment:
If the GH repo is public, should we just omit the access flag altogether? If the flag is not set at publish time, does it inherit access settings from the npm repo?

Reply:
If it's not set it looks like it'll either be "restricted" if it's a scoped package or public if it's unscoped, and setting it here will change the existing setting (useful for going private -> public or public -> private, although going public -> private will obviously not be fully guaranteed to cover everything)

Reply:
Apparently this flag does nothing on existing npm repos
https://docs.npmjs.com/cli/v8/commands/npm-publish#access

> Note: Using the --access flag on the npm publish command will only set the package access level on the initial publish of the package. Any subsequent npm publish commands using the --access flag will not have an effect to the access level. To make changes to the access level after the initial publish use npm access.